include/openspace/performance/performancemanager.h

```diff
@@ -58,14 +58,40 @@ class PerformanceManager {
     void storeIndividualPerformanceMeasurement(std::string identifier, long long nanoseconds);
     void storeScenePerformanceMeasurements(const std::vector<SceneGraphNode*>& sceneNodes);
     
-    PerformanceLayout* performanceData();
+    void outputLogs();
+
+    void writeData(std::ofstream& out, const std::vector<float> data);
+
+    const std::string formatLogName(std::string nodeName);
+
+    void logDir(std::string dir);
+    std::string logDir();
+    void prefix(std::string prefix);
+    std::string prefix();
+
+    void enableLogging();
+    void disableLogging();
+    void toggleLogging();
+    void setLogging(bool enabled);
+    bool loggingEnabled();
 
+    PerformanceLayout* performanceData();
```

Comment:
This wasn't a change I made, and it looks like it's passed around and used in a non-`const` fashion on a couple places. Maybe worth coming back to, but could mean a more significant rewrite.